sections/03_categorize.md

```diff
@@ -1,23 +1,43 @@
-## Does deep learning create a strategic inflection point in how we categorize individuals with respect to health and disease?
```

Comment:
Thanks - sorry for that - I think I shouldn't have merged into my own master before things made it into greenelab/master